### PR TITLE
Feature/2

### LINKED ISSUE #2: Custom vendor name
**To do:**
- Provide functionality that enables user to use it's own vendor name instead of Snowdog

Solution: 

Add additional prompt question to https://github.com/SnowdogApps/create-alpaca-theme/blob/master/src/create-alpaca-theme.js#L57 and use as user **BASE_PATH**

### PR DESCRIPTION
**Pull request by patrykbura**

### CODE REVIEW

File: templates/frontools/themes.json.sample [hunk 1]
```diff
@@ -10,8 +10,8 @@
     ]
   },
   "YOUR_THEME_NAME": {
-    "src": "app/design/frontend/Snowdog/YOUR_THEME_NAME",
-    "dest": "pub/static/frontend/Snowdog/YOUR_THEME_NAME",
+    "src": "app/design/frontend/YOUR_VENDOR/YOUR_THEME_NAME",
+    "dest": "pub/static/frontend/snowdog/YOUR_THEME_NAME",
```
Comment (ofrankowska) [inline templates/frontools/themes.json.sample]: I think it should be YOUR_VENDOR
from frontools docs:

> dest - full path to pub/static/[theme_area]/[theme_vendor]/[theme_name]

File: src/constants/constants.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ export const PACKAGE_PATH = {
 }
 
 // BASE PATHS
-export const BASE_PATH = 'app/design/frontend/Snowdog/'
+export const BASE_PATH = 'app/design/frontend/'
```
Comment (ofrankowska) [inline src/constants/constants.js]: base path meaning isn't obvious to me. what do you think about including theme in the name?

**Review by ofrankowska — CHANGES_REQUESTED** [review]

File: src/constants/constants.js (hunk 2 above)
Comment (patrykbura) [reply src/constants/constants.js]: ok

File: templates/frontools/themes.json.sample (hunk 1 above)
Comment (patrykbura) [reply templates/frontools/themes.json.sample]: Has to stay **Snowdog**

**Review by ofrankowska — APPROVED** [review]

**Pull request merged**